op-chain-ops/cmd/celo-migrate/ancients.go

```diff
@@ -104,56 +94,67 @@ func migrateAncientsDb(ctx context.Context, oldDBPath, newDBPath string, batchSi
 
 func readAncientBlocks(ctx context.Context, freezer *rawdb.Freezer, startBlock, endBlock, batchSize uint64, out chan<- RLPBlockRange) error {
 	defer close(out)
-
 	for i := startBlock; i < endBlock; i += batchSize {
-		count := min(batchSize, endBlock-i+1)
+		count := min(batchSize, endBlock-i)
 		start := i
 
-		blockRange := RLPBlockRange{
-			start:    start,
-			hashes:   make([][]byte, count),
-			headers:  make([][]byte, count),
-			bodies:   make([][]byte, count),
-			receipts: make([][]byte, count),
-			tds:      make([][]byte, count),
-		}
-		var err error
-
-		blockRange.hashes, err = freezer.AncientRange(rawdb.ChainFreezerHashTable, start, count, 0)
+		blockRange, err := loadAncientRange(freezer, start, count)
```

Comment:
@piersy This is the only refactor to the migration code path from the audited code that I haven't reverted. It's straightforward enough that I thought it might be okay to keep, since we need to use loadAncientRange in the continuity script. Happy to revert it though if you think that's best